integration/test_auth.py

```diff
@@ -372,6 +373,37 @@ def test_import_users_with_password(api_key):
     finally:
         auth.delete_user(uid)
 
+@pytest.fixture
+def action_code_settings():
+    return auth.ActionCodeSettings('http://localhost')
+
+def _validate_link_url(link):
+    assert isinstance(link, six.string_types)
+    six.moves.urllib.parse.urlparse(link)
+
+def test_password_reset(new_user_with_params):
+    link = auth.generate_password_reset_link(new_user_with_params.email)
+    _validate_link_url(link)
```

Comment:
We should also probably do a password reset using the generated link and verify it works. If you're not sure how to do that, we can implement that bit later in a future PR.